draft-ietf-jsonpath-base.md

```diff
@@ -1087,13 +1098,15 @@ Notes:
 
 The following table lists filter expression operators in order of precedence from highest (binds most tightly) to lowest (binds least tightly).
 
+<!-- FIXME: Should the syntax column be split between unary and binary operators? -->
+
 | Precedence | Operator type | Syntax |
 |:--:|:--:|:--:|
 |  5  | Grouping | `(...)` |
 |  4  | Logical NOT | `!` |
 |  3  | Relations | `==`&nbsp;`!=`<br>`<`&nbsp;`<=`&nbsp;`>`&nbsp;`>=`<br>`=~`<br>` in ` |
 |  2  | Logical AND | `&&` |
-|  1  | Logical OR | `\|\|` |
+|  1  | Logical OR | `¦¦`   |
```

Comment:
The symbol looks wrong to me. It appears to consist of two punctured vertical bars (`¦¦`, each of which is U+00A6) rather than vertical bars (`||`, each of which is U+007C).

Reply:
That is a markdown trick.  The backslashes are definitely wrong. ¦¦ is rendered || by xml2rfc.